docs/pages/checkbox.js

```diff
@@ -23,48 +37,493 @@ export default function CheckboxPage({ generatedDocGen }: {| generatedDocGen: Do
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use Checkbox at the start of a table row to make it clear which rows are multi-selectable"
+            defaultCode={`
+function Example() {
+  const [checked1, setChecked1] = React.useState(false);
+  const [checked2, setChecked2] = React.useState(false);
+  const [checked3, setChecked3] = React.useState(false);
+
+  return (
+    <Table accessibilityLabel="Campaign selection" maxHeight={200}>
+      <Table.Header sticky>
+        <Table.Row>
+          <Table.HeaderCell/>
+          <Table.HeaderCell>
+            <Text weight="bold">Active</Text>
+          </Table.HeaderCell>
+          <Table.HeaderCell>
+            <Text weight="bold">Name</Text>
+          </Table.HeaderCell>
+        </Table.Row>
+      </Table.Header>
+      <Table.Body>
+        <Table.Row>
+          <Table.Cell>
+            <Box width={20}>
+              <Checkbox
+                checked={checked1}
+                id="Summertime_picnic_2"
+                onChange={({ checked }) => setChecked1(checked)}
+                size="sm"
+              />
+            </Box>
+          </Table.Cell>
+          <Table.Cell>
+            <Switch
+              onChange={() => {}}
+              id="disabled"
+              switched={false}
+            />
+          </Table.Cell>
+          <Table.Cell>
+            <Label htmlFor="Summertime_picnic_2">
+              <Text>Summertime picnic</Text>
+            </Label>
+          </Table.Cell>
+        </Table.Row>
+        <Table.Row>
+          <Table.Cell>
+            <Box width={20}>
+              <Checkbox
+                checked={checked2}
+                id="Summer_1950_2"
+                onChange={({ checked }) => setChecked2(checked)}
+                size="sm"
+              />
+            </Box>
+          </Table.Cell>
+          <Table.Cell>
+            <Switch
+              onChange={() => {}}
+              id="Summer_1950"
+              switched
+            />
+          </Table.Cell>
+          <Table.Cell>
+            <Label htmlFor="Summer_1950_2">
+              <Text>Summer 1950</Text>
+            </Label>
+          </Table.Cell>
+        </Table.Row>
+        <Table.Row>
+          <Table.Cell>
+            <Box width={20}>
+              <Checkbox
+                checked={checked3}
+                id="Back_to_school_2"
+                onChange={({ checked }) => setChecked3(checked)}
+                size="sm"
+              />
+            </Box>
+          </Table.Cell>
+          <Table.Cell>
+            <Switch
+              onChange={() => {}}
+              id="Back_to_school"
+              switched={false}
+            />
+          </Table.Cell>
+          <Table.Cell>
+            <Label htmlFor="Back_to_school_2">
+              <Text>Summer 1950</Text>
```

Comment:
I think this one is wrong, causing an a11y failure

Reply:
DONE